src/Defra.PTS.Web.Application/Validation/PetKeeperNameValidator.cs

```diff
@@ -13,7 +13,7 @@ public PetKeeperNameValidator(IStringLocalizer<SharedResource> localizer)
 
         When(x => !string.IsNullOrWhiteSpace(x.Name), () =>
         {
-            RuleFor(x => x.Name).MaximumLength(AppConstants.MaxLength.PetKeeperName).WithMessage($"Enter your full name, using {AppConstants.MaxLength.PetKeeperName} characters or less");            
+            RuleFor(x => x.Name).MaximumLength(AppConstants.MaxLength.PetKeeperName).WithMessage(x => localizer[$"Enter your full name, using {AppConstants.MaxLength.PetKeeperName} characters or less"]);            
             RuleFor(x => x.Name).Matches(AppConstants.RegularExpressions.Name).WithMessage("Enter a name using only letters, hyphens or apostrophes");
```

Comment:
Do you have a localized version of this error?

Reply:
Removed reg ex entirely